samples/system-test/vms_api.test.js

```diff
@@ -1,33 +0,0 @@
-/**
```

Comment:
Why are you deleting this file? Samples should have tests, and this one works.

Reply:
The sample is not obsolete so long as it is shown on cloud.google.com. It's used here: https://cloud.google.com/compute/docs/tutorials/nodejs-guide#gettingstarted-google-api-client

Reply:
This sample has nothing to do with this library. It says
```js
var google = require('googleapis');
var compute = google.compute('v1');
```
so it's basically a sample for [google-api-nodejs-client](https://github.com/google/google-api-nodejs-client), not for `@google-cloud/compute`.

The other two samples are correct:
```js
const Compute = require('@google-cloud/compute');
```

Reply:
Regardless, it cannot be deleting without breaking that page, unless we copy the sample to another location, update the page to point to the new location, and _then_ delete the sample from this repo.

Reply:
The test for this sample actually never worked (at least in CI). I fixed the sample (to use the latest syntax) and the test, it works now, can you take a look? Thanks!

Reply:
Correction: it never worked because of the wrong NPM script, not because the test was wrong.